build/Build.cs

```diff
@@ -313,7 +313,7 @@ private AbsolutePath[] GetDockerProjects()
             DotNetBuild(s => s
                 .SetProjectFile(Solution)
                 .SetConfiguration(Configuration)
-                .SetAssemblyVersion(version)
```

Comment:
So what is the value of the assembly version in this case then?

Reply:
`FileVersion` is what I actually needed to set. Assembly version if not set comes from the project's Assembly version property. And if the `FileVersion` is not set it taken from the value set in `AssemblyVersion`  - `AssemblyInfo.cs`
![image](https://user-images.githubusercontent.com/8049896/157841759-8f931edd-0ed5-46ef-8247-d19a41662fb5.png)